integration-tests/tests/src/tests/serialization.ts

```diff
@@ -89,49 +75,17 @@ const testSetups: TestSetup[] = [
         // Playlists
         const p1 = realm.create<IPlaylistNoId>(PlaylistSchemaNoId.name, {
           title: "Playlist 1",
-          songs: [
-            s1,
-            s2,
-            s3,
-            {
-              artist: "Unique artist 1",
```

Comment:
I had to delete non-pre-created objects since those do not seem to be supported right now.. it does make these tests less different from one another though so I am not sure multiple playlist objects are needed (also it makes no difference whether primary keys are supplied or not in functionality now, so maybe I should remove that too).

Reply:
> non-pre-created objects

What do you mean by that?